src/app/components/turnos/gestor-agendas/operaciones-agenda/panel-agenda.component.ts

```diff
@@ -76,40 +73,26 @@ export class PanelAgendaComponent implements OnInit {
         }
     }
 
-    eliminarProfBaja() {
-        const profesionales = [];
-        if (this.agenda.profesionales) {
-            for (let i = 0; i < this.agenda.profesionales.length; i++) {
-                this.servicioProfesional.get({ id: this.agenda.profesionales[i].id }).subscribe(resultado => {
-                    if (resultado[0].habilitado) {
-                        profesionales.push(this.agenda.profesionales[i]);
-                    }
-                });
-            };
-            this.agenda.profesionales = profesionales;
-        }
-    }
-
     guardarAgenda(agenda: IAgenda) {
 
         if (this.alertas.length === 0) {
             // Quitar cuando esté solucionado inconveniente de plex-select
             let profesional = [];
-            if (this.agenda.profesionales && this.agenda.profesionales.length > 40) {
+            if (this.editaAgendaPanel.profesionales && this.editaAgendaPanel.profesionales.length > 40) {
```

Comment:
Se podría simplificar la condición por: this.editaAgendaPanel.profesionales?.length > 40

Reply:
cambié el nombre en la declaración y no me di cuenta 😄 gracias!